cfgov/unprocessed/apps/owning-a-home/js/form-explainer.js

```diff
@@ -200,15 +200,15 @@ class FormExplainer {
    * current focus.
    */
   openAndScrollToExpandable( imageOverlay, targetExpandable ) {
-    let targetExpandableTarget = targetExpandable.querySelector(
+    const targetExpandableTarget = targetExpandable.querySelector(
       '.o-expandable_target'
     );
 
     targetExpandableTarget.focus();
 
     window.setTimeout( () =>
       targetExpandableTarget.click(),
-      100
+    100
```

Comment:
This change isn't needed. Did the linter pick this up?

Reply:
yeah, that was the auto fix

Reply:
Oh wait, nevermind. I thought both lines were in the arrow function. Maybe do this instead:
```
window.setTimeout( 
  () => targetExpandableTarget.click(),
  100
);
```